### ISSUE TITLE
Is there a plan to add some text matching models?

### ISSUE THREAD

**Issue by fierceX** [dmlc/gluon-nlp#616]
Similar to https://github.com/NTMC-Community/MatchZoo

**Comment by kenjewu** [comment]
@fierceX Are you planning to add some model code and training scripts?

**Comment by fierceX** [comment]
@kenjewu I need these models for my work, I might add some models, but it shouldn't be too much.
Are you interested in this?

**Comment by kenjewu** [comment]
@fierceX Yes, I am interested in this :smile: . I have written some related models.

**Comment by szha** [comment]
@fierceX @kenjewu this looks very useful! Let me know if you need a feature branch for you to start tinkering.

**Comment by eric-haibin-lin** [comment]
@fierceX any update on this?

**Comment by fierceX** [comment]
@eric-haibin-lin I may prepare some models in the near future. Need to open a branch?

**Comment by szha** [comment]
@fierceX feel free to create a branch if that helps
